src/context/json.ts

```diff
@@ -1,14 +1,16 @@
-import { ResponseTransformer } from '../response'
+import { ResponseTransformer, MockedResponse } from '../response'
 
 /**
  * Sets the given Object as the JSON body of the response.
  * @example
- * res(json({ foo: 'bar' }))
+ * res(json({ key: 'value' }))
  */
-export const json = (body: Record<string, any>): ResponseTransformer => {
+export const json = <BodyType>(
```

Comment:
The way this works is that it inherits the type of the given argument (`body`) and returns a `ResponseTransformer` with the response body annotated with that inferred type.

Combine this with the `res` response composition function that infers the type from the request handler itself, and if the response transformer type from the response composition doesn't match a response transformer type inferred from the context utility's value, you get a TypeScript error.

Reply:
very good 🥇

Reply:
very good 🥇